be/src/env/env_s3.cpp

```diff
@@ -128,7 +128,7 @@ class S3ClientFactory {
     // (where as before 1.8 it has been hard coded default of "us-east-1").
     // Part of that change is looking through the ec2 metadata, which can take a long time.
     // For more details, please refer https://github.com/aws/aws-sdk-cpp/issues/1440
-    static Aws::Client::ClientConfiguration s_config;
+    // static Aws::Client::ClientConfiguration s_config;
```

Comment:
Ignore this part because of issue: https://github.com/StarRocks/starrocks/issues/3927. Will be fixed later.